lib/methods/query.js

```diff
@@ -34,7 +34,24 @@ exports.query = function (Model, options, callback) {
 		whereQuery = this.translateWhereToQuery(where, values);
 	}
 
-	var order = Model.translateKeysForPayload(options.order);
+	/**
+	 * Transform stirng property to default orderTo clause
+	 * @param {any} prop
+	 * @returns {Object}
+	 */
+	function propToOrder(prop) {
+		var order = {};
+		order[prop] = 1;
+		return order;
+	}
+
+	var order = _([options.order])
+		.map(function (orderBy) {
```

Comment:
Yeah, just wanted to make it more functional, it could reduce the bugs :) What do you think, should I leave this approach for now?